include/numerics/parsed_function_parameter.h

```diff
@@ -24,11 +24,12 @@
 // Local Includes -----------------------------------
 #include "libmesh/libmesh_common.h"
 #include "libmesh/parameter_accessor.h"
-#include "libmesh/parsed_function.h"
 
 namespace libMesh
 {
 
+template <typename T> class ParsedFunction;
```

Comment:
Because it's not being used directly in the library. We use it in GRINS, which is why I caught it --- it didn't compile. :)